package.json

```diff
@@ -36,7 +36,9 @@
     "posttest": "npm run lint",
     "test:dev": "lb-mocha --allow-console-logs DIST/test/**/*.js && npm run posttest",
     "prestart": "npm run build",
-    "start": "node .",
+    "start": "concurrently --kill-others \"npm run start:app\" \"npm run start:recommender\"",
```

Comment:
Why not `npm run start:recommender && npm run start:app`?

Reply:
Because then the start process hangs at `npm run start:recommeder` as the server is running ad doesn't return a response to the command line to allow it to continue to the next `npm` script. 

We used `concurrently` in https://github.com/strongloop/loopback4-example-microservices as well.

Reply:
Another option: https://www.npmjs.com/package/npm-run-all